packages/nodejs/src/server/onMessage.ts

```diff
@@ -1,22 +1,25 @@
 import WebSocket from 'ws';
 import { describe, run } from './messageHandlers'
 import { MessageHandler } from './MessageHandler';
-import { Application } from '@data-story/core';
+import { Application, NullStorage } from '@data-story/core';
+import { getItems } from './messageHandlers/getItems';
 
 export const onMessage = async (
   ws: WebSocket,
   message: string,
   app: Application,
+  storage: NullStorage
```

Comment:
Since we now are using the Storage to put items in it, it is no longer a NullStorage, we should consider renaming it to something better, maybe InMemoryStorage?